src/store/aggregator/sagas/depositTokens.ts

```diff
@@ -88,12 +92,19 @@ export function* depositTokens({ payload }: AggregatorActions['submit']) {
 
   let submitEffect: SagaContractEffect;
 
-  if (isRSK) {
+  if (!isCrossChain) {
+    let minimumRewardNumber = rewardAmount * (100 - payload.slippageSlider) / 100;
+    minimumRewardNumber = minimumRewardNumber > 0 ? minimumRewardNumber : 0;
+    const minimumReward = utils.parseUnits(
+      minimumRewardNumber.toString(),
+      DEFAULT_ASSET_DECIMALS
+    );
     submitEffect = call(
-      massetContract.mintTo,
+      massetContract.mintToWithMinimumReward,
```

Comment:
RUSDT -> XUSD conversions are failing with "reward under minimum" error:
![image](https://github.com/BabelFishProtocol/babelfish-dapp/assets/765825/48904989-3874-4872-aa81-e191c77c7f22)

https://dashboard.tenderly.co/tx/rsk-testnet/0x777438c456bc7da900cf188ade77392b9d685c07e5dde727a98313306a6c272d

this was working previously, but I just noticed that in my successful test prior to this that I never actually received the reward amount I was shown (should have got ~1.0006 XUSD):
https://explorer.testnet.rsk.co/tx/0xf7a0313164a4ba39664d897b73811e3a8c630b18b2fcd46601da35ee411f9913?__ctab=Token%20Transfers